README.md

```diff
@@ -47,21 +71,30 @@ This will be the file that holds your S3 and SQS configurations.
       * a first-time login with your GitHub credentials
       * in the Settings -> Applications -> federalist view in your GitHub Account
 1. Type `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` in terminal.
-1. Visit https://login.fr.cloud.gov/passcode to get a one time passcode.
-1. Enter your passcode back into terminal.
-1. Type `cf apps`.
+1. Visit https://login.fr.cloud.gov/passcode to get a one time passcode.  You'll need to login using your cloud.gov credentials.
+1. Enter your passcode back into terminal.  (Note: if you have a problem at this point it's likely because you don't have permissions.  [Contact us] (federalist-support@gsa.gov) for help.)
```

Comment:
Again, not sure we want to offer email support like this. I'd suggest that these instructions are targeted at 18F/TTS folks who are working on Federalist (so maybe that should be started at the start of this local development section).

Reply:
Agreed; we shouldn't put instructions for 18F/TTS in here. Everything should apply to someone who wants to run their own indepedent Federalist instance.

Reply:
Unfortunately because of the tight coupling to the cloud.gov/AWS services, 18F/TTS folks working on Federalist will need these steps. I think we should prioritize their use of these instructions since we really want to make getting a local instance setup for a newly onboarded person as easy as possible.

Maybe we could just prefix these cloud.gov steps with `**For 18F/TTS developers:**`?

Reply:
That sounds good to me.

Reply:
editing **For 18F/TTS developers:** in.